src/dataviews/dataview-model-base.js

```diff
@@ -256,6 +274,10 @@ module.exports = Model.extend({
   },
 
   update: function (attrs) {
+    if (_.has(attrs, 'source')) {
+      var message = '[SOURCE] Source present in UPDATE attrs. ' + JSON.stringify(attrs.source);
```

Comment:
I'm sure you're aware, but just in case... We'll get two error messages in this case: one when `update` is invoked, and another one when `change:source` will be triggered. Nothing bad with it...

Reply:
I know, but we can receive the `update` message and not the `change` one if the source is the same.

Reply:
:+1: